content/en/agent/guide/private-link.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Set up a AWS PrivateLink from AWS us-east-1 region to Datadog
+kind: guide
+further_reading:
+- link: "agent/logs"
+  tag: "Documentation"
+  text: "Enable log collection with the Agent."
+- link: "/integrations/amazon_web_services/#set-up-the-datadog-lambda-function"
+  tag: "Documentation"
+  text: "Collect logs from your AWS services"
+---
+
+<div class="alert alert-info">
+The PrivateLink setup is only available to <b>forward logs and metrics to Datadog in AWS us-east-1 region</b>. Reach out to <a href="/help">Datadog Support</a> to enable this feature for your organization.
+</div>
+
+This guide walks you through how to configure [AWS PrivateLink][1] for use with Datadog. AWS PrivateLink simplifies the security of data shared with Datadog by eliminating the exposure of data to the public Internet.
+
+## Overview
+
+The overall process consist in configuring an internal endpoint in your VPC that local Datadog Agents can send data to. Your VPC endpoint is then peered with the endpoint within Datadog VPC.
+
+{{< img src="agent/guide/private_link/vpc_schema.png" alt="VPC Schema" >}}
+
+## Create your VPC endpoint
+
+1. Connect to the AWS console and create a new VPC endpoint:
+    {{< img src="agent/guide/private_link/create_vpc_endpoint.png" alt="Create VPC endpoint" style="width:60%;" >}}
+2. Select **Find service by name**.
+3. Fill the *service name* text box with: `com.amazonaws.vpce.us-east-1.vpce-svc-<UNIQUE_ID>` where `<UNIQUE_ID>` is the ID communicated by the Datadog [Support team][2]:
+    {{< img src="agent/guide/private_link/vpc_service_name.png" alt="VPC service name" style="width:60%;" >}}
+4. Hit the *verify* button.
+    **Note**: If it does not return *Service name Found*, reach out to [Datadog support team][2] as this means that your AWS account id was not authorised to connect on the Datadog VPC service endpoint.
```

Comment:
does this button really return "Service name Found" with that exact capitalization?

(also, capitalize `id` to `ID`)